samples/Gallery/Pages/StylesPage.fs

```diff
@@ -28,6 +72,9 @@ module StylesPage =
 
                 TextBlock("I'm just a text")
 
+                AutoCompleteBox([])
+                    .watermark("I'm an AutoCompleteBox styled to have a crimson watermark and accept Return/Enter")
+                    .inlineStyles(coloredTextBoxWatermark(Brushes.Crimson), acceptReturnOnAutoCompleteTextBox())
```

Comment:
The recommended ways of styling :
1. Using modifiers

```fsharp
    let textBlockStyle (this: WidgetBuilder<'msg, IFabTextBlock>) =
        this
            .verticalAlignment(VerticalAlignment.Center)
            .foreground(SolidColorBrush(Colors.Black))
```
2. Load `xaml` styles

```fsharp
.styles([ "avares://Gallery/Styles/TextStyles.xaml" ])
```

I would be ok adding support for `inlineStyles`. One disadvantage is that Fabulous will recreate the whole `Style` during the view diffing as oppose to a single element property(See 1).